lib/Proxy.js

```diff
@@ -173,8 +177,7 @@ define([
 
 		_publishInSequence: function (message) {
 			var session = this._getSession(message.sessionId);
-
-			if (message.sequence <= session.lastSequence) {
+			if (!isFatalErrorFromPreExecutor(message) && message.sequence <= session.lastSequence) {
```

Comment:
Yes, exactly. If you have any suggestions on how to implement this in a better way, please let me know :)